pkg/okteto/context.go

```diff
@@ -55,6 +56,7 @@ const (
 
 var (
 	CurrentStore *OktetoContextStore
+	reg          = regexp.MustCompile("[^A-Za-z0-9]+")
```

Comment:
do we really need this as global var?

Reply:
In that way the regex is compiled one once instead of each time it has to be used

Reply:
makes sense!